security.txt

```diff
@@ -0,0 +1,5 @@
+Contact: email security at lists.ilias.de

```

Comment:
I don't think this meets the specification. Shouldn't it start with `mailto:` in case of an email address?
```
Contact: mailto:security@lists.ilias.de
```
Source: https://www.rfc-editor.org/rfc/rfc9116#name-contact